src/components/animated-menu-button/animated-menu-button.js

```diff
@@ -218,6 +219,7 @@ class AnimatedMenuButton extends React.Component {
         data-element='close'
         key='close'
         onClick={ this.closeHandler }
+        onKeyPress={ this.handleCloseIconKeyPress }
```

Comment:
Maybe we should change this to an anchor instead? That way `onClick` should work for click and pressing enter out of the box.

Reply:
sorry - a `button` should do the trick - not an `anchor`

Reply:
Done